install_cluster.py

```diff
@@ -0,0 +1,260 @@
+import os
+import shutil
+import string
+import random
+from os.path import expanduser, exists, join
+
+import yaml
+
+from common import (CfyNode, copy, get_dict_from_yaml,
+                    JUMP_HOST_CONFIG_PATH, JUMP_HOST_DIR,
+                    JUMP_HOST_LICENSE_PATH, JUMP_HOST_SSH_KEY_PATH,
+                    logger, move, run, sudo)
+
+INSTANCES_TYPES = ['postgresql', 'rabbitmq', 'manager']  # Order is important
+CERT_PATH = '{0}/.cloudify-test-ca'.format(expanduser('~'))
+RPM_NAME = 'cloudify-manager-install.rpm'
+CONFIG_FILES = 'config_files'
+
+JUMP_HOST_RPM_PATH = join(JUMP_HOST_DIR, RPM_NAME)
+LOCAL_INSTALL_CLUSTER_DIR = join(JUMP_HOST_DIR, 'cluster_install')
+LOCAL_CERTS_DIR = join(LOCAL_INSTALL_CLUSTER_DIR, 'certs')
+LOCAL_CONFIG_DIR = join(LOCAL_INSTALL_CLUSTER_DIR, CONFIG_FILES)
+
+REMOTE_INSTALL_CLUSTER_DIR = join('/tmp', 'cluster_install')
+REMOTE_CERTS_DIR = join(REMOTE_INSTALL_CLUSTER_DIR, 'certs')
+REMOTE_RPM_PATH = join(REMOTE_INSTALL_CLUSTER_DIR, RPM_NAME)
+REMOTE_CONFIG_DIR = join(REMOTE_INSTALL_CLUSTER_DIR, CONFIG_FILES)
+
+
+def _generate_instance_certificate(instance):
+    run(['cfy_manager', 'generate-test-cert', '-s',
+         instance.private_ip+','+instance.public_ip])
+    move(join(CERT_PATH, instance.private_ip+'.crt'),
+         join(CERT_PATH, instance.name+'_cert.pem'))
+    new_key_path = join(CERT_PATH, instance.name+'_key.pem')
+    move(join(CERT_PATH, instance.private_ip+'.key'), new_key_path)
+    sudo(['chmod', '444', new_key_path])
+
+
+def _generate_certs(instances_dict):
+    logger.info('Generating certificates')
+    for instances_list in instances_dict.values():
+        for instance in instances_list:
+            _generate_instance_certificate(instance)
+    copy(join(CERT_PATH, 'ca.crt'), join(CERT_PATH, 'ca.pem'))
+    os.mkdir(LOCAL_CERTS_DIR)
+    copy(CERT_PATH+'/.', LOCAL_CERTS_DIR)
```

Comment:
Looks like this repo wants flake8's attention.

Reply:
Thanks, I'll add CircleCI tests as well to avoid these things :)